Foundation/NSPropertyList.swift

```diff
@@ -117,6 +138,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return true
     } else if input === kCFBooleanFalse {
         return false
+    } else if CFGetTypeID(input) == _CFKeyedArchiverUIDGetTypeID() {
```

Comment:
CFGetTypeID assumes that the input is a CFTypeRef; non CF backed objects will potentially read past the allocations so we should use the property `_cfTypeID` instead